CVQM.md

```diff
@@ -0,0 +1,79 @@
+
+Every original / processed video sequence pair is run through two main steps. First, the requested calibration is run, and results are saved in a file named after the processed video file, with '_calibration.txt' appended. Second, the requested model is run, and results are saved in a file named after the processed video file, with '_model.txt' appended.
+
+Any errors that occur are recorded in a file named after the processed video file, with '_errors.txt' appended. If that file is absent, no errors occurred. Each line on this file will start with a number, indicating error type, and then a string describing the error. 
+Type 1 errors are fatal data input/output issues (e.g., operation cannot continue due to invalid input argument, file read error, or file write error). Type 2 errors are calibration warnings (e.g., still sequence, temporal registration failure, spatial shift beyond search limits). Type 3 errors are non-fatal warnings.
+
+## VQM Model Options
+The following VQM models options are available:
+
+* 'none'- No model will be calculated - run calibration only (see below).
+* 'general' - NTIA General Model, as given in the ANSI T1.801.03-2003, ITU-T J.144 (03/04), ITU-R BT.1683 (06/04), and [TR-02-392](https://its.ntia.gov/publications/details?pub=2423). 
+* 'developers' - Developer's model, a fast running variant of the General Model, described in [TR-02-392](https://its.ntia.gov/publications/details?pub=2423)
+* 'lowbw' - Low Bandwidth Model, described [here](https://its.ntia.gov/publications/details?pub=2575)
+* 'fastlowbw' - Fast Low Bandwidth Model, a fast running variant of the low bandwidth model 
+
+The CVQM software provides a clean implementation of these VQM models. 
+It is intended to be ported into other programming languages or to be called from other software. 
+
+## Calibration Options
+The following calibration options are available. 
+Technical details are available in [TR-02-392](https://its.ntia.gov/publications/details?pub=2423) and [TR-06-433b](https://its.ntia.gov/publications/details?pub=2465).
+
+### 'none'
+No calibration will be performed. Assume that the first frame in original and processed video files align temporally. Run model with default calibration values which assumes that the processed video file is perfectly calibrated.
+
+### 'manual' 
+Read the calibration file created on a previous run. The values on the beginning of each line may be manually modified.
+
+### 'rrcal' 
+Perform reduced reference calibration as given in ntia_tr_06_433a.pdf (except assume no spatial scaling). These algorithms use random processes, which may yield slightly different results from one run to another. 
```

Comment:
Please capitalize NTIA TR

Reply:
I corrected all of these references to TR-06-433 and updated them from 433a to 433b.